curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md

```diff
@@ -121,8 +121,8 @@ You should have a `p` element below your first `h2` element.
 
 ```js
 assert.strictEqual(
-  document.querySelector('h2')?.nextElementSibling,
-  document.querySelectorAll('p')?.[1]
+  document.querySelector('h2')?.nextElementSibling.tagName,
+  document.querySelectorAll('p')?.[1].tagName
```

Comment:
```suggestion
  document.querySelector('h2')?.nextElementSibling.tagName, 'P'
```

Wouldn't something like this be better? Same with the test three before this one (I can't suggest changes there on GH).

Reply:
Yeah, this makes more sense. And maybe we should change the other test too @zairahira

Reply:
@Dario-DC,
I was able to find one more similar test and I have updated it.